milestones/serializers.py

```diff
@@ -0,0 +1,67 @@
+"""
+Data layer serialization operations.  Converts querysets to simple
+python containers (mainly arrays and dicts).
+"""
+from . import models
+
+
+def serialize_milestone(milestone):
+    """
+    Milestone object-to-dict serialization
+    """
+    return {
+        'id': milestone.id,
+        'name': milestone.name,
+        'namespace': milestone.namespace,
+        'description': milestone.description
+    }
+
+
+def serialize_milestone_with_course(course_milestone):
+    """
+    CourseMilestone serialization (composite object)
+    """
+    return {
+        'id': course_milestone.milestone.id,
+        'name': course_milestone.milestone.name,
+        'namespace': course_milestone.milestone.namespace,
+        'description': course_milestone.milestone.description,
+        'course_id': course_milestone.course_id
+    }
+
+
+def serialize_milestone_with_course_content(course_content_milestone):
+    """
+    CourseContentMilestone serialization (composite object)
+    """
+    return {
+        'id': course_content_milestone.milestone.id,
+        'name': course_content_milestone.milestone.name,
+        'namespace': course_content_milestone.milestone.namespace,
+        'description': course_content_milestone.milestone.description,
+        'course_id': course_content_milestone.course_id,
+        'content_id': course_content_milestone.content_id
+    }
+
+
+def serialize_milestones(milestones):
```

Comment:
I ran into some confusion at the onset of the project trying to work with DRF serializers outside of the context of an HTTP request, so I went with the RYO (roll yer own) approach and haven't had time to get back around to it.  I think  we could note this as an area of tech debt and address in the next cycle, unless there's a good reason to use DRF serialization here?